### PR TITLE
Assignment 1 completed.

### PR DESCRIPTION
**Pull request by FilipeMalachias**

### CODE REVIEW

File: src/main/java/ca/ciccc/Assignment1.java [hunk 1]
```diff
@@ -89,7 +114,19 @@ public static String bmi(int kgs, double meters) {
...
+        float kph = (meters / 1000.0f) / (timeSeconds / 3600.0f);
+        float mph = kph / 1.609f;
+
+        DecimalFormat df = new DecimalFormat("#.####");
+        df.setRoundingMode(RoundingMode.DOWN);
+
+
+
+        String result = String.format("Your speed in meters/second is %s\nYour speed in km/h is %s\nYour speed in miles/h is %s", df.format(mps), df.format(kph), df.format(mph));
+        return result;
```
Comment (nator333) [inline src/main/java/ca/ciccc/Assignment1.java]: For your information,

```
String.format("Your speed in meters/second is %1$s\nYour speed in km/h is %2$s\nYour speed in miles/h is %3$s", df.format(mps), df.format(kph), df.format(mph));
```

is also possible and sometimes easier to see 👍

File: src/main/java/ca/ciccc/Assignment1.java [hunk 2]
```diff
@@ -128,6 +180,23 @@ public static String powers(int number) {
      */
     public static String arithmetic(int a, int b) {
 
-        return "";
+        int firstInteger = a;
+        int secondInteger = b;
+        int sum = firstInteger + secondInteger;
+        int dif = firstInteger - secondInteger;
+        int mult = firstInteger * secondInteger;
+        float med = ( (firstInteger + secondInteger) / 2);
```
Comment (nator333) [inline src/main/java/ca/ciccc/Assignment1.java]: You need to convert ints to floats before the calculation, I suppose.
Dose it work?

File: src/main/java/ca/ciccc/Assignment1.java (hunk 1 above)
Comment (itzderr) [reply src/main/java/ca/ciccc/Assignment1.java]: 👍

File: src/main/java/ca/ciccc/Assignment1.java (hunk 2 above)
Comment (itzderr) [reply src/main/java/ca/ciccc/Assignment1.java]: `float med = ((float)(firstInteger + secondInteger) / 2);`

**Pull request closed without merging**